### PR TITLE
fix: Added missing error message

### LINKED ISSUE #597: Error message for missing all fields under update user profile.
When we delete all fields in input, an appropriate error message should be showned and listed under Responses.
**To Reproduce**

1. Go to 'User-update user profile'.
2. Provide authorization access token.
3.  Input playload:
 {
  }
4.  Click on 'Execute'
5. Scroll down to 'Respense.'
6. 
Screenshot (72)

**Expected behavior**
That error message should be listed in the responses.

**Desktop (please complete the following information):**
 - OS: Windows
 - Browser chrome

### PR DESCRIPTION
**Pull request by devkapilbansal**
Error message for missing all fields in update user profile is added.

Fixes #597

### Type of Change:

- Quality Assurance
- Documentation

**Code/Quality Assurance Only**
- This change requires a documentation update (software upgrade on readme file)

### How Has This Been Tested?
Run `python -m unittest discover tests`

ss

### Checklist:

- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials
- [x] I have made corresponding changes to the documentation

**Code/Quality Assurance Only**

- [x] My changes generate no new warnings 
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by devkapilbansal** [comment]
@PrashanthPuneriya @faznan3nazer please review it

File: app/api/resources/user.py [hunk 1]
```diff
@@ -129,7 +129,14 @@ def get(cls):
     @users_ns.doc("update_user_profile")
     @users_ns.expect(auth_header_parser, update_user_request_body_model)
     @users_ns.response(HTTPStatus.OK, "%s" % messages.USER_SUCCESSFULLY_UPDATED)
-    @users_ns.response(HTTPStatus.BAD_REQUEST, "%s" % messages.NAME_INPUT_BY_USER_IS_INVALID)
+    @users_ns.response(
+        HTTPStatus.BAD_REQUEST,
+        "%s\n%s"
+        % (
+            messages.NAME_INPUT_BY_USER_IS_INVALID,
```
Comment (PrashanthPuneriya) [inline app/api/resources/user.py:136]: I think `NO_DATA_FOR_UPDATING_PROFILE_WAS_SENT` message is enough there is no need of `NAME_INPUT_BY_USER_IS_INVALID`. Am I missing something else @devkapilbansal ?

**Review by PrashanthPuneriya — DISMISSED** [review]
LGTM!

**Review by vj-codes — DISMISSED** [review]
LGTM 🎉

File: app/api/resources/user.py [hunk 2]
```diff
@@ -129,7 +129,14 @@ def get(cls):
...
+            messages.NO_DATA_FOR_UPDATING_PROFILE_WAS_SENT
+        )
```
Comment (isabelcosta) [inline app/api/resources/user.py]: Can you please use f-strings for this? we have some issues especially for converting these parts of the code to use f-strings :)
```
f"{messages.NAME_INPUT_BY_USER_IS_INVALID}\n"
f"{messages. NO_DATA_FOR_UPDATING_PROFILE_WAS_SENT}\n"
```

**Review by isabelcosta — CHANGES_REQUESTED** [review]
@devkapilbansal I just left a small change request. I can approve after you address it :)

File: app/api/resources/user.py (hunk 2 above)
Comment (devkapilbansal) [reply app/api/resources/user.py]: @isabelcosta please review it once again

**Review by isabelcosta — APPROVED** [review]
👏 Great work @devkapilbansal 🎉

**Review by vj-codes — APPROVED** [review]

**Comment by isabelcosta** [comment]
The changes made in this PR were tested on https://ms-backend-review-pr-925.herokuapp.com/. Following are the results:

1. Code review - Done

2. All possible responses (positive and negative tests) were tested as below:

  * _Run app and check Swagger documentation_  
    _Screenshot/gif_:  
image

    _Expected Result_:  message "NO_DATA_FOR_UPDATING_PROFILE_WAS_SENT" should show up on Swagger UI
    _Actual Result_:  message does show up in SwaggerUI

3. Status of PR Changed to: Ready to Merge.

**Comment by isabelcosta** [comment]
@devkapilbansal thank you so much for this contribution 🤗

**Pull request merged**